persistence/soft-index/src/main/java/org/infinispan/persistence/sifs/EntryHeader.java

```diff
@@ -7,26 +7,33 @@
  */
 public class EntryHeader {
    static final int MAGIC = 0xBE11A61C;
-   static final boolean useMagic = false;
-   static final int HEADER_SIZE = 28 + (useMagic ? 4 : 0);
+   //no magic neither no internal metadata
+   static final int OLD_HEADER_SIZE = 24;
+   static final int HEADER_SIZE = 32;
```

Comment:
`HEADER_SIZE_10_1` and `HEADER_SIZE_11_0` would be more informative.

Reply:
ack